server/controllers/user-auth.js

```diff
@@ -0,0 +1,47 @@
+
+import { hash } from 'bcrypt';
+import client from '../database/db_connect';
+
+export async function usersignup(req, resp) {
+    const {lastname , firstname , email , type} = {
+        lastname : req.body.lastname,
+        firstname : req.body.firstname,
+        email : req.body.email,
+        type : req.body.type
+    }
```

Comment:
this should be short like 
`const {lastname, firstname, email, type} = req.body`